addons/website_sale/views/templates.xml

```diff
@@ -850,8 +850,14 @@
     </template>
 
     <template id="website_sale.products_categories_list" active="True" name="eCommerce Categories">
+        <!--
+            We must keep `t-attf-class` attr. for both following divs to work correctly with
+            products_categories_list_collapsible and option_collapse_products_categories
+        -->
+        <div t-attf-class="d-contents">
```

Comment:
I thought this was a bootstrap class; could you change the name to clarify that it's a placeholder?